### PR TITLE
French translation (WIP)

### PR DESCRIPTION
**Pull request by dalink**
Also some english translation.
This is a work in progress.

### CODE REVIEW

File: app/views/catarse_bootstrap/explore/index.html.slim [hunk 1]
```diff
@@ -23,8 +23,8 @@
 
   .w-container.u-marginbottom-10
     .u-text-center.u-marginbottom-40
-      a.explore-toggle.fontsize-larger.link-hidden-white.fontweight-light href="#" 
-        | Explore projetos incríveis &nbsp;
+      a.explore-toggle.fontsize-larger.link-hidden-white.fontweight-light href="#"
+        | Voir les projets à financer &nbsp;
```
Comment (adrianob) [inline app/views/catarse_bootstrap/explore/index.html.slim]: This string should be translated.

**Comment by dalink** [comment]
I have made the correction.
Do you some kinds of best practices concerning translation or evolution for the Catarse platform ?
I'm preparing a plateform in France and I'd like to had some feature like Twitter or Linkedin login, payment provider ...

In first time, I'll will made a full french translation, in second time I'll add some modules.

File: app/views/catarse_bootstrap/explore/index.html.slim [hunk 2]
```diff
@@ -35,15 +35,15 @@
             .w-col.w-col-2.w-col-small-6.w-col-tiny-6
               a.explore-link.w-inline-block.btn-category [
                 class="#{'double-line' if category.to_s.size > 13}"
-                href="#by_category_id/#{category.id}" 
-                data-name=(params[:locale] == 'en' ? category.name_en : category.name_pt)
+                href="#by_category_id/#{category.id}"
+                data-name=( (category.name_fr if params[:locale] == 'fr') || (category.name_en if params[:locale] == 'en') || category.name_pt )
```
Comment (adrianob) [inline app/views/catarse_bootstrap/explore/index.html.slim]: Let's make this more generic and move the logic to a category decorator. Something like:
def display_name
I18n.available_locales.include?(params[:locale].to_sym) ? source.send('name_'+params[:locale]) : source.name_pt
end
Not actually tested this but you get the idea...

**Comment by dalink** [comment]
I'have me the decorator.
I have seen that this decorator may be use in some other place. It's in my Todo list.

File: db/seeds.rb [hunk 3]
```diff
@@ -3,72 +3,76 @@
...
      name_en: name[:en]
    })
+   category.update_attributes({
+     name_fr: name[:fr]
+   })
  end
 
 {
-  company_name: 'Catarse',
+  company_name: 'BuzzEtBoost',
```
Comment (devton) [inline db/seeds.rb]: Can you rollback to catarse instead BuzzEtBoost? 

[]'s

**Comment by dalink** [comment]
Done

**Comment by adrianob** [comment]
Looks good  :+1: 

Do you plan on adding more commits or can I merge this already?

**Comment by dalink** [comment]
Yes, you can merge

**Pull request merged**